.github/workflows/ci.yaml

```diff
@@ -15,6 +15,9 @@ jobs:
     container: lampepfl/dotty:2020-04-24
 
     steps:
+      - name: Set up JDK version
```

Comment:
I would call this step "Set JDK 11 as default" so we can easily know which version we're setting.